application/src/main/resources/ecc.yml

```diff
@@ -53,10 +53,17 @@ connection:
     ## Allow routing requests directly to a remote datacenter.
     ## This allows locks for other datacenters to be taken in that datacenter instead of via the local datacenter.
     ## If clients are prevented from connecting directly to Cassandra nodes in other sites this is not possible.
-    ## If remote routing is disabled its not possible to use LOCAL_SERIAL consistency for the locking,
+    ## If remote routing is disabled and consistency serial uses the DEFAULT mode its not possible to use LOCAL_SERIAL consistency for the locking,
     ## instead SERIAL consistency will be used for those request.
     ##
     remoteRouting: true
+    ##
+    # consistencySerial value can be DEFAULT, which means that the serial consistency for lightweight transactions is based
+    # on the remoteRouting configurations. It is possible to define as LOCAL or SERIAL as well, the first one defines serial consistency
+    # as LOCAL_SERIAL which requires a quorum of replicas in the local DC, also SERIAL defines a serial consistency of SERIAL
```

Comment:
I don't think we need to explain what LOCAL_SERIAL and SERIAL is. Instead, we should explain the implications, i.e if you use remoteRouting: false and LOCAL_SERIAL here then all locks will be taken locally in DC. I.e There's a risk that multiple nodes in different datacenters will be able to lock the same nodes -> multiple repairs on the same range/node at the same time.

Reply:
Done